webapp/src/components/MeetingQuickView.tsx

```diff
@@ -0,0 +1,182 @@
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Alert,
+  Link as MuiLink,
+  Stack,
+  TextField,
+  Typography,
+} from '@mui/material';
+import CheckCircleOutlineIcon from '@mui/icons-material/CheckCircleOutline';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import { Link as ReactRouterLink } from 'react-router-dom';
+import { LoadingButton } from '@mui/lab';
+import React, { FormEventHandler, useContext, useState } from 'react';
+
+import { EntityId, Meeting as APIMeeting } from '../types/api';
+import { formatDate, formatTime } from '../helpers/dateTime';
+import SessionContext from './SessionContext';
+import useApiData from '../helpers/useApiData';
+
+interface MeetingQuickViewProps {
+  meeting: APIMeeting;
+}
+
+const MeetingQuickView = ({ meeting }: MeetingQuickViewProps) => {
+  const { principalId } = useContext(SessionContext);
+  const [changedMeetingNoteIds, setChangedMeetingNoteIds] = useState<
+    EntityId[]
+  >([]);
+  const [isExpanded, setIsExpanded] = useState(false);
+  const [isSavingMeetingNote, setIsSavingMeetingNote] = useState(false);
+  const [isSuccessIndicated, setIsSuccessIndicated] = useState(false);
+  const [meetingNoteText, setMeetingNoteText] = useState('');
+  const [saveMeetingNoteError, setSaveMeetingNoteError] = useState(null);
+
+  const { data: meetingWithNotes, error } = useApiData<APIMeeting>({
```

Comment:
It caught me off-guard that this component both takes a `Meeting` as a prop, but also fetches the meeting again from the API. I can think of a number of way where this could be made more intuitive.

The problem is that this use of `useApiData` begs the question, "Doesn't this component already have all the meeting data? Why is it fetching the meeting data again?"

Reply:
The `meeting` prop does not have any meeting notes.  The implementation is incomplete here and should do the second fetch only when the accordion is expanded.  

Could fetch the all meetings with all of their individual notes and pass all of the data down fromm the Parent component in order to only do one fetch call.  That would require a change to the `MeetingsService`.

Reply:
We had had a discussion about using the `deps` and `shouldFetch` arguments to the `useApiData` hook to address this. It appears that that was not implemented. Has that approach been discounted as a solution?

Reply:
No, I'll work on that solution.